templates/interstitial.html

```diff
@@ -23,4 +23,17 @@ <h1 class="question__title">{{ block.content.title }}</h1>
     {% set title = _("If you can’t answer questions for this person") %}
     {% include 'partials/individual-response-guidance.html' %}
   {% endif %}
+
 {% endblock form_content %}
+
+{% block submit_button %}
+    {{
+        onsButton({
+            "text": continue_button_text | default(_("Save and continue")),
+            "classes": "u-mt-l",
```

Comment:
shoudl this be `l` or `xl`, it is `xl` for questions.

Reply:
@jrbarnes9 ?

Reply:
It's been reduced to `l` to account for the bottom margin on last element of the content